### ISSUE TITLE
java.lang.AbstractMethodError: abstract method "io.reactivex.Observable rx_activity_result2.OnPreResult.response(int, int, android.content.Intent)"

### ISSUE THREAD

**Issue by laysdra7265** [miguelbcr/RxPaparazzo#70]
When I run Example code
```java

        RxPaparazzo.single(this)
                .usingGallery()
                .subscribeOn(Schedulers.io())
                .observeOn(AndroidSchedulers.mainThread())
                .subscribe(new Consumer<Response<ChatActivity, FileData>>() {
                    @Override
                    public void accept(Response<ChatActivity, FileData> response) throws Exception {
                        // See response.resultCode() doc
                    /*if (response.resultCode() != RESULT_OK) {
                        response.targetUI().showUserCanceled();
                        return;
                    }

                    response.targetUI().loadImage(response.data());*/
                    }
                });
```

then get this exception
looks like the RxAtivityResult2 has update something but here is outdated.

**Comment by miguelbcr** [comment]
Hi @laysdra7265 !
I'll take a look this week. 
If you figure out something let me know.
Thanks!

**Comment by laysdra7265** [comment]
For work around ,I fix it by use the older version of RxActivityResult.
`compile 'com.github.VictorAlbertos:RxActivityResult:0.4.0-2.x'`
look like the latest version of RxActivityResult change the
`io.reactivex.Observable rx_activity_result2.OnPreResult.response(int, android.content.Intent)`
to
`io.reactivex.Observable rx_activity_result2.OnPreResult.response(int, int, android.content.Intent)`
so in the `com/miguelbcr/ui/rx_paparazzo2/interactors/PickFile.java` should update the code of anonymous class of `OnPreResult`
in `private OnPreResult getOnPreResultProcessing()`

```java
 private OnPreResult getOnPreResultProcessing() {
    return new OnPreResult() {
      @Override
      //Problem here
      public Observable<Uri> response(int responseCode, @Nullable final Intent intent) {
        if (responseCode == Activity.RESULT_OK && intent != null && intent.getData() != null) {

          Uri pickedUri = intent.getData();
          PermissionUtil.grantReadPermissionToUri(targetUi, pickedUri);

          return Observable.just(intent.getData());
        } else {
          return Observable.empty();
        }
      }
    };
  }
```

**Comment by miguelbcr** [comment]
I have no time this week, sorry. I'll try to fix it next week.
Anyway you are free to make a PR.
Thank you!

**Comment by onomated** [comment]
Any intention of upgrading this to the latest RxActivityResult version soon?  Or would you rather leave this supporting the current version?

**Comment by miguelbcr** [comment]
Hi all!,
sorry, I've been very busy last month. 😞 
I hope to do it today, anyway PR are welcome.

**Comment by onomated** [comment]
No problem.  Definitely understand.  If you can't get to it today, let me know and I'll take a stab at getting it all updated.  If you have any process/procedures I should know about on developing in this codebase please let me know.

**Comment by miguelbcr** [comment]
Hi @laysdra7265 @onomated !
sorry for the late.
I've released [version 0.5.3-2.x](https://github.com/miguelbcr/RxPaparazzo/releases/tag/0.5.3-2.x)
Let me know if it works for you

**Comment by onomated** [comment]
Works great! Tested from camera image capture, gallery images, remote images from Google drive etc.  All good on my end.
Thanks a lot for a great component @miguelbcr !
